jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -856,18 +848,20 @@ pipeline {
                                 inputManifest: "manifests/${INPUT_MANIFEST}",
                                 platform: 'windows',
                                 architecture: 'x64',
-                                distribution: "zip",
+                                distribution: 'zip',
                                 continueOnError: params.CONTINUE_ON_ERROR,
                                 incremental: params.INCREMENTAL,
                                 previousBuildId: params.PREVIOUS_BUILD_ID
                             )
                             String buildManifestUrl = buildManifestObj.getUrl(JOB_NAME, BUILD_NUMBER)
+                            String buildManifestUrlOpenSearch = [buildManifestObj.getArtifactRootUrl(JOB_NAME_OPENSEARCH, 'latest'), 'builds', 'opensearch', 'manifest.yml'].join('/')
```

Comment:
Just wondering can these be global vars?

Reply:
No they cannot be since each stage has its own build manifest URL.

Reply:
Since you are using the same var name, we sure these vars are local vars for that specific stage?

Reply:
Yes! There are passed as method args https://github.com/opensearch-project/opensearch-build/pull/4374/files#diff-3f7479e22eeeedaf481e3de253091fff48673c90624a9e1101a57924864d9b8eR1025